app/models/APIModel.scala

```diff
@@ -1,3 +1,4 @@
 package models
 
 case class Status(service: String, environment: String, serverTime: String)
+case class Product(name: String, quantity: Int, price: Double)
```

Comment:
I don't believe this is being used. If this is intended to match the database schema for doobie, replace the db file model declaration